zwave/rootfs/etc/services.d/zwave/run

```diff
@@ -19,6 +20,14 @@ if bashio::config.has_value 'instance'; then
     OZW_INSTANCE=$(bashio::config 'instance')
 fi
 
+# Set custom config_dir and user_dir
+if bashio::config.has_value 'config_dir'; then
+    OZW_CONFIG=$(bashio::config 'config_dir')
+fi
+if bashio::config.has_value 'user_dir'; then
+    OZW_USER=$(bashio::config 'user_dir')
+fi
```

Comment:
We should not do this.

We don't want this into the Home Assistant configuration folder. 

This feature isn't implemented yet, because it awaits:

https://github.com/home-assistant/supervisor/issues/1842

Reply:
It by default writes to "/data/ozw/config". It only changes when user wants to. 
If /config can cause conflict with HA, can we let user chose to use /share folder? (It is the user repurposing the folder, not the addon)

Reply:
Well, the same answer as before, this feature isn't implemented because of the linked RFC.

Reply:
I'm not sure I understand what you said unimplemented "feature".
The config.json -> map allows me to access these folders. If I map `config` and/or `share` in config.json, user can set the flags here to ask ozwdaemon to use those folders, so the other addons (e.g., samba, ssh) can access them.

Reply:
For feasibility, I just want to clarify that this is tested working - I am testing my zwave device support xml in /config

Reply:
This feature wasn't added because of the linked issue/RFC. This is not possible at this time. We don't want this configuration to end up in these folders you are suggesting; nor do we want this configurable.

As said, this feature isn't added before because of the given reasoning. Please remove that feature from this PR.